sql/src/main/java/io/crate/executor/transport/TableCreator.java

```diff
@@ -40,71 +41,78 @@
 import org.elasticsearch.action.admin.indices.template.put.PutIndexTemplateResponse;
 import org.elasticsearch.action.admin.indices.template.put.TransportPutIndexTemplateAction;
 import org.elasticsearch.cluster.ClusterService;
+import org.elasticsearch.common.logging.ESLogger;
+import org.elasticsearch.common.logging.Loggers;
+import org.elasticsearch.common.settings.Settings;
 import org.elasticsearch.indices.IndexAlreadyExistsException;
 import org.elasticsearch.indices.IndexTemplateAlreadyExistsException;
 
-import java.util.List;
 import java.util.Locale;
-import java.util.UUID;
 
-public class CreateTableTask extends AbstractChainedTask {
+public class TableCreator {
```

Comment:
could probably be a singleton that has all dependencies injected and takes the createTableStatement as argument to the start method